packages/terra-menu/src/MenuItem.module.scss

```diff
@@ -67,12 +67,17 @@
     }
   }
 
+  .is-top-level .text {
+    margin-right: var(--terra-menu-item-text-margin-right, 0);
```

Comment:
You'll need a unique theme variable here.

Reply:
Fixed with b912c82.